daemon/graphdriver/graphtest/graphtest_unix.go

```diff
@@ -181,6 +181,12 @@ func DriverTestCreateEmpty(t *testing.T, drivername string) {
 		t.Fatal(err)
 	}
 
+	defer func() {
+		if err := driver.Remove("empty"); err != nil {
```

Comment:
Oh, lol, meant to comment on the other one below :D
It's just a nit, just felt like a bit cleaner and less DRY